src/pages/api/configs/tryToggleEdit.tsx

```diff
@@ -0,0 +1,33 @@
+import Consola from 'consola';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+function Post(req: NextApiRequest, res: NextApiResponse) {
+  const { tried } = req.body;
+  // Try to match the password with the EDIT_PASSWORD env variable
+  if (tried === process.env.EDIT_MODE_PASSWORD) {
+    process.env.DISABLE_EDIT_MODE = process.env.DISABLE_EDIT_MODE === 'true' ? 'false' : 'true';
+    return res.status(200).json({
+      success: true,
+    });
+  }
+  // Warn that there was a wrong password attempt (date : wrong password, person's IP)
+  Consola.warn(
+    `${new Date().toLocaleString()} : Wrong edit password attempt, from ${
+      req.headers['x-forwarded-for']
+    }`
+  );
+  return res.status(200).json({
+    success: false,
+  });
+}
+
+export default async (req: NextApiRequest, res: NextApiResponse) => {
+  // Filter out if the request is a POST or a GET
```

Comment:
Comment is not needed

Reply:
It's a copy-paste of the login password API page that's why it's there

Reply:
Okay, remove it on the password page too

Reply:
It's unrelevant to the PR. but i could merge both methods to one API route try password with query parameters to avoid code duplication

Reply:
Yes, that would be good IMO. It would also make sense to add rate limiting to this route...

It's up to you, if you want to merge or work on this in a separate PR